### ISSUE TITLE
`manual_uwnrap_or_default` does not detect `match` with `block` in first arm.

### ISSUE THREAD

**Issue by MKowalski8** [software-mansion/cairo-lint#301]
### Problem

As in title, it does not detect cases like the following
``` cairo
fn main() {
  let x: Option<[u64; 5]> = Option::Some([1, 2, 3, 4, 5]);

  match x {
    Option::Some(v) => {
      v
    },
    Option::None => {
      [0; 5]
    }
  };
}
```

However, if the first arm were written without a block, like `Option::Some(v) => v`, the lint would be correctly triggered.

### Scarb Version

```text
2.11.4
```

### Notes

The other cases in [this function](https://github.com/software-mansion/cairo-lint/blob/5b4f0c810ba23cb4a41b326507ec00b5aa6e5b57/src/lints/manual/mod.rs#L50) might suffer from the same issue, so it’s worth investigating them as well.
